bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -55,6 +55,10 @@ spec:
       containers:
       - name: sdn
         image: {{.SDNImage}}
+        lifecycle:
+          preStop:
+            exec:
+              command: ["rm","-f","/etc/cni/net.d/80-openshift-network.conf"]
```

Comment:
there is already a `preStop` handler at the end of the container spec that does this

Reply:
fixed.